docs/release/release_0_6_0.md

```diff
@@ -55,25 +64,136 @@ number), now is a good time to undo the workarounds for newer versions of
 napari! If you run into any issues please get in touch [on GitHub
 issues](https://github.com/napari/napari) or on our [Zulip chat room](https://napari.zulipchat.com)!
 
-- Implement command palette widget ([#5483](https://github.com/napari/napari/pull/5483))
-- Fix issues displaying polygons with holes in Shapes ([#6654](https://github.com/napari/napari/pull/6654))
+On the user space, we now offer several options to orient the axes any way you
+like:
+
+1. **Through the camera API:** the `Viewer.camera` instance gains two new
+  attributes: `orientation`, and `orientation2d`, which is just the last two
+  dimensions of `orientation`. You can set the direction that the *depth*,
+  *vertical*, and *horizontal* axes point to, respectively in that order, as
+  follows ([#7663](https://github.com/napari/napari/pull/7663)):
+
+  ```python
+  # 2D
+  viewer.camera.orientation2d = ('up', 'right')
+  # 3D
+  viewer.camera.orientation = ('away', 'up', 'right')
+  ```
+
+  See an example of this in action in
+  {ref}`sphx_glr_gallery_xarray-latlon-timeseries.py`.
+
+2. By right clicking on the dimension toggle in the viewer, and setting the
+  axis orientations using the drop-down menus
+  ([#7686](https://github.com/napari/napari/pull/7686)), which in 3D will
+  further indicate whether the resulting coordinate frame is [right-handed or
+  left-handed](https://en.wikipedia.org/wiki/Right-hand_rule)
+  ([#7770](https://github.com/napari/napari/pull/7770)):
+
+  ![axis orientation dialog](https://github.com/user-attachments/assets/f73898ec-9156-4f73-ab7f-ee2a7cc17fe1)
+
+3. If you want to use a specific axis orientation consistently, you can set the
```

Comment:
Not a blocker of this alpha, but for the final release we should make clear what the settings were before 0.6.0 vs. the new default, so that a user who wants to go back to the old behavior can do so relatively easily.
See: https://github.com/napari/napari/pull/7787#issuecomment-2794549343